contentcuration/contentcuration/frontend/channelList/views/KCard/BaseCard.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div
+    class="card"
+    tabindex="0"
+    @focus="cardFocus"
+    @hover="cardHover"
+  >
+    <div
+      :class="$computedClass({ ':focus': $coreOutline })"
+    >
+      <li class="remove-list-style">
```

Comment:
According to the spec, I think this should be the outer most layer, for accessibility reasons